libs/hscim/package.yaml

```diff
@@ -69,6 +69,8 @@ dependencies:
   -  network-uri          >= 2.6.2 && < 2.7
   -  servant              >= 0.16.2 && < 0.19
   -  servant-server       >= 0.16.2 && < 0.19
+  -  servant-client       >= 0.16.2 && < 0.19
+  -  servant-client-core  >= 0.16.2 && < 0.19
```

Comment:
package.yaml is not stack, but the source file we're generating the cabal file from.  the cabal file needs version bounds in this case because i need to publish it to hackage.